modules/samples/consumers/object-detection-service/src/main/webapp/index.jsp

```diff
@@ -0,0 +1,133 @@
+<html>
```

Comment:
Fixed in https://github.com/hemikak/product-cep/commit/6210d26b510acc5fc699d74e24cdcd17d3c8dd1d